### PR TITLE
Add hubot-welcome to welcome users to the channel

### PR DESCRIPTION
**Pull request by bobsilverberg**
This PR is now ready to test. The bot is in the channel as webqabot-review-app. If he goes away you can wake him up by going to https://webqabot-pr-20.herokuapp.com/

Info on hubot-welcome can be found at https://github.com/bobsilverberg/hubot-welcome

@davehunt r?

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
       "required": true
     },
     "HUBOT_GITHUB_EVENT_NOTIFIER_ROOM": {
-      "required": true
+      "value": "#webqa-test"
```
Comment (davehunt) [inline app.json]: Is this so that once we change the value for our bot, review apps still use the test channel?

**Comment by davehunt** [comment]
r+

File: app.json (hunk 1 above)
Comment (bobsilverberg) [inline app.json]: Exactly. I assumed we wanted that, for testing purposes.

**Comment by bobsilverberg** [comment]
Thanks for the review, @davehunt.

**Pull request merged**